content/workshops/advanced-css.md

```diff
@@ -12,10 +12,6 @@ tags:
   - CSS
   - Sass
 events:
-  - venue: Webconf.asia
-    url: https://2020.webconf.asia/
-    adr: Hong Kong
-    date: 2020-08-28
```

Comment:
@mirisuzanne Should this be updated on their website as well? https://2020.webconf.asia/workshop-css

Reply:
Possibly? I'm never sure how to handle when I change the name of something, but it's the same general material. Also not sure how important it is to update a conference that I expect to have cancelled due to COVID? Open to thoughts on both.